src/coordinator/users.js

```diff
@@ -510,6 +510,33 @@ class Users {
       `${this._baseUrl}/users/${userId}/stacks/${stackId}`
     );
   }
+
+  /**
+   * Syncs the user's roles and application access with auth0
+   *
+   * API Endpoint: '/users/:userId/sync'
+   * Method: GET
+   *
+   * @param {string} userId The ID of the user
+   *
+   * @returns {Promise}
+   * @fulfill {undefined}
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.coordinator.users
+   *   .syncWithAuth0('36b8421a-cc4a-4204-b839-1397374fb16b')
+   *   .catch((err) => console.log(err));
+   */
+  syncWithAuth0(userId) {
```

Comment:
Let's leave out Auth0 from name, some thoughts:
- sync
- syncWithAuthProvider
- syncPermissions

Reply:
This is good and I take back my approval

Reply:
But y tho?

This endpoint get's deleted the moment auth0 isn't used anymore.

Reply:
Mostly because it is an implementation detail.

Other reasons:

1. auth0 could change
2. code stays around longer that we want
3. if there isn't a good reason to change, we can agree with the API when the call is 1:1 to an API